js/templates/modals/settings/advanced/server.html

```diff
@@ -0,0 +1,45 @@
+<form class="box padKids padStack clrP clrBr js-server">
+  <div class="flexRow">
+    <h3><%= ob.polyT('settings.server') %></h3>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label for="currentFirewallType">
+        <%= ob.polyT('settings.currentFirewallType') %>
+      </label>
+    </div>
+    <div class="col9">
```

Comment:
Okay. I think semantics-wise it makes sense to be a readonly form slot if it comes from the same model as the other slots in the form. Also, I get that semantics of rendered HTML is not a main concern here. In any case, I wasn't that clear on what the firewall requirement meant. And now I see mentioned above maybe a firewall slot is being removed.